source/slang/slang-ast-support-types.h

```diff
@@ -485,6 +494,7 @@ namespace Slang
         const ReflectClassInfo* m_superClass;       ///< The super class of this class, or nullptr if has no super class. 
         const char* m_name;                         ///< Textual class name, for debugging 
         CreateFunc m_createFunc;                    ///< Callback to use when creating instances
+        DestroyFunc m_destroyFunc;                  ///< Called to inplace destroy (ie not backing memory) of this type
```

Comment:
The lack of symmetry between the `Create` and `Destroy` cases here is a bit subtle and unfortunate, but probably not worth addressing at this point.